docs/fastdds/transport/whitelist.rst

```diff
@@ -36,8 +39,32 @@ For example:
       :lines: 2-3,5-
       :append: </profiles>
 
+* Interface whitelist filled with interface names:
+
+.. tabs::
+
+  .. tab:: C++
+
+    .. literalinclude:: /../code/DDSCodeTester.cpp
+      :language: c++
+      :start-after: //WHITELIST-NAME
+      :end-before: //!--
+      :dedent: 8
+
+  .. tab:: XML
+
+    .. literalinclude:: /../code/XMLTester.xml
+      :language: xml
+      :start-after: <!-->WHITELIST-NAME
+      :end-before: <!--><-->
+      :lines: 2-3,5-
+      :append: </profiles>
+
+.. important::
+
+  The values in this list should match the IP addresses or interface names on your machine for the corresponding networks. If none of the values in the whitelist match the interfaces on your machine, all the interfaces in the whitelist are filtered out and an invalid interface is assigned.
```

Comment:
```suggestion
  If none of the values in the transport descriptor's whitelist match the interfaces on your machine, all the interfaces in the whitelist are filtered out and no communication will be established through that transport.
```

Reply:
Just to shorten it a bit, and saying we add an invalid ip is perhaps too low level for a user.